bin/make/frank-plugin.sh

```diff
@@ -95,6 +101,7 @@ xcrun xcodebuild build \
   ARCHS="i386 x86_64" \
   VALID_ARCHS="i386 x86_64" \
   ONLY_ACTIVE_ARCH=NO \
+  EFFECTIVE_PLATFORM_NAME="-iphonesimulator" \
```

Comment:
Starting in Xcode 7.2, simulator binaries (apps and libraries) are being built to `Debug-iphoneos` and not to `Debug-iphonesimulator`.   After exploring a bunch of other build settings, I found that this was the missing argument.